pyproject.toml

```
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "epidemics-suppression"
+version = "0.1.0"
+description = ""
+authors = ["Andrea Maiorana, Marco Meneghelli"]
```

```suggestion
authors = ["Andrea Maiorana", "Marco Meneghelli"]
```

Consider adding emails if you don't fear spam